app/controller.js

```diff
@@ -0,0 +1,9 @@
+const { fetchCategories } =  require('./models');
+
+
+exports.getCategories = (request, response) => {
+fetchCategories().then((categories) => { 
+    response.status(200).send({categories: categories})
+})
```

Comment:
It's just missing a .catch() block to deal with any errors which may happen asynchonrously